cyclonedx_buildroot/_internal/cli.py

```diff
@@ -174,22 +174,21 @@ def run(*, argv: Optional[Sequence[str]] = None, **kwargs: Any) -> Union[int, No
     br_bom = Bom()
     br_bom.metadata.component = rootComponent = Component(name=args.product_name,
                                                           version=args.product_version)
-    # TODO Specification 1.6 ends support of "manufacture". Remove in future release.
-    br_meta = BomMetaData(manufacture=OrganizationalEntity(name=args.manufacturer_name),
-                          manufacturer=OrganizationalEntity(name=args.manufacturer_name),
+
+    br_meta = BomMetaData(manufacturer=OrganizationalEntity(name=args.manufacturer_name),
                           component=rootComponent)
 
     br_bom.metadata = br_meta
```

Comment:
get rid of this `br_bom.metadata = br_meta`.
you are overriding the metadata 5 lines after assigning a value to `br_bom.metadata.component`.  -- so you are actually dropping the root component right after adding it ...

Reply:
That is some old old crusty bad code. Thank you for finding it. I will correct the code flow.